pandas/_libs/internals.pyx

```diff
@@ -378,25 +378,23 @@ def get_blkno_indexers(int64_t[:] blknos, bint group=True):
 
         object blkno
         object group_dict = defaultdict(list)
-        int64_t[:] res_view
 
     n = blknos.shape[0]
-
-    if n == 0:
-        return
-
+    result = list()
```

Comment:
I think there are some "bugs" in Cython with generator functions that were yielding build warnings. I opened an issue on the tracker for it here:

https://github.com/cython/cython/issues/3430

But for now figure OK to just convert to a list instead of a generator func anyway